### PR TITLE
Specify key value in timeseries reporting

### LINKED ISSUE #435: Specify key value in timeseries outputs
For surface variables, enable the `TimeseriesCSVExport` measure to output specific surfaces, instead of the default all (`*`).

### PR DESCRIPTION
**Pull request by aspeake**
Resolves #435 

## Pull Request Description
Allows users to specify a key value when requesting timeseries csvs, for example:
`output_variables: Surface Outside Face Incident Solar Radiation Rate per Area|Surface 2` outputs the variable for just Surface 2, instead of every surface.

## Checklist

Not all may apply:

- [x] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [x] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [x] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [x] All tests are passing (green) on circleci
- [x] The [changelog](https://github.com/NREL/OpenStudio-BuildStock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with master

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: measures/TimeseriesCSVExport/measure.rb [hunk 1]
```diff
@@ -98,7 +98,14 @@ def energyPlusOutputRequests(runner, user_arguments)
     results = output_meters.create_custom_building_unit_meters
 
     output_vars.each do |output_var|
-      results << OpenStudio::IdfObject.load("Output:Variable,*,#{output_var},#{reporting_frequency};").get
+      if output_var.include? '|'
+        key_val = output_var.split('|')[1]
+        var_name = output_var.split('|')[0]
```
Comment (joseph-robertson) [inline measures/TimeseriesCSVExport/measure.rb:103]: I'd go with: `output_var, key_value = output_var_item.split("|")`

Comment (aspeake) [reply measures/TimeseriesCSVExport/measure.rb]: addressed

**Comment by joseph-robertson** [comment]
@aspeake I think we probably want to test this using local docker or eagle.

**Comment by aspeake** [comment]
@joseph-robertson 

Branch has been tested on Eagle with `project_testing` and timeseries outputs work as expected. The original timeseries csv export inputs behave as they did previously. I think it is probably ready for review + merge

**Pull request merged**